src/emmy/calculus/derivative.cljc

```diff
@@ -441,17 +442,17 @@
 ;; instances.
 
 (def D
-  "Derivative operator. Takes some function `f` and returns a function
-  whose value at some point can multiply an increment in the arguments, to
-  produce the best linear estimate of the increment in the function value.
+  "Derivative operator. Takes some function `f` and returns a function whose value
+  at some point can multiply an increment in the arguments to produce the best
+  linear estimate of the increment in the function value.
 
-  For univariate functions, [[D]] computes a derivative. For vector-valued
```

Comment:
It's a pedantic point whether a row vector is a Jacobian or not, but I do think that univariate was correct here. A scalar valued function of 2 variables gives you a down tuple when differentiated, and a univariate vector valued function gives you an up tuple, which I think of as a derivative because you can write it as $f'$